frontend-public/src/actionCreators/varianceActionCreator.js

```diff
@@ -9,28 +9,29 @@ import { ENVIRONMENT } from "@/constants/environment";
 import { createRequestHeader } from "@/utils/RequestHeaders";
 import CustomAxios from "@/utils/customAxios";
 
-export const createVariance = ({ mineGuid }, payload) => (dispatch) => {
+export const createVariance = (mineGuid, mineName, payload) => (dispatch) => {
```

Comment:
Switched it mainly because we aren't following this pattern in any of our actionCreators.
Happy to chat about the benefits of passing an object vs positional params though!